package.json

```diff
@@ -52,10 +52,24 @@
           "type": "boolean",
           "default": true,
           "description": "If true, a line will be drawn under the highlighted characters"
+        },
+        "vimFindHighlight.enableAutoHighlight": {
+          "type": "boolean",
+          "default": true,
+          "description": "If true, the extension will automatically highlight the characters that can be reached with the f motion"
         }
       }
     },
-    "commands": []
+    "commands": [
+      {
+        "command": "vimFindHighlight.toggleAuthoHighlight",
+        "title": "Toggle Auto Highlight"
+      },
+      {
+        "command": "vimFindHighlight.highlightCharacters",
+        "title": "Highlight Characters"
+      }
```

Comment:
Let's maybe prefix the titles with the name of the extension to make it easier to search for its commands. Other extensions do this as well. Check screenshots.

```suggestion
      {
        "command": "vimFindHighlight.toggleAuthoHighlight",
        "title": "Vim Find Highlight: Toggle Auto Highlight"
      },
      {
        "command": "vimFindHighlight.highlightCharacters",
        "title": "Vim Find Highlight: Highlight Characters"
      }
```

<img width="602" alt="Screenshot 2024-08-31 at 11 41 02" src="https://github.com/user-attachments/assets/7a0d2360-9e10-4dbc-810c-438726a3ca63">
<img width="602" alt="Screenshot 2024-08-31 at 11 41 11" src="https://github.com/user-attachments/assets/38817e38-3eea-4fb2-b324-88b23139693a">

Reply:
Ah yes! Somehow I thought vscode would do that automagically! Will do!